docs/user-guide/environment-variables.md

```diff
@@ -54,6 +54,7 @@ _Note: Environment variables set in one job cannot be accessed in another job. T
 | SD_TEMPLATE_NAMESPACE | Namespace of the template the job is using (e.g.: `d2lam`; blank if not using template) |
 | SD_TEMPLATE_VERSION | Version of the template the job is using (blank if not using template) |
 | SD_TOKEN | JWT token for the build |
+| SD_SCHEDULED_BUILD | The build is triggered by scheduler or not |
```

Comment:
I prefer that it refers like [SD_SONAR_ENTERPRISE](https://github.com/screwdriver-cd/guide/pull/422/files#diff-6ab6f3a2f928a8d91f94784b2ac898d1dae84a062968b5e2d09e9e169f3ffc02R80)